qa/suites/fs/fscrypt/tasks/1-tests/fscrypt-common.yaml

```diff
@@ -1,3 +1,10 @@
+overrides:
+  install:
+    extra_system_packages:
+      rpm:
+        - fscrypt
+      deb:
+        - fscrypt
```

Comment:
As I remembered in some old distros the `fscrypt` package couldn't be found and installed. But for the latest distros we use in qa should be fine.

Let's just keep this in mind.